docs/dev-tools/zombienet/overview.md

```diff
@@ -111,13 +115,12 @@ In order to install Zombienet, there are multiple options available, depending o
     npm run zombie -- -p native spawn host-current-files/minimal.toml
     ```
 
-    !!! warning
-        The command above mounts the current directory to the `/workspace` directory inside the Docker container. This allows Zombienet to access the configuration file and other files in the current directory. If you want to mount a different directory, replace `$(pwd)` with the desired directory path.
+    The command above mounts the current directory to the `/workspace` directory inside the Docker container. This allows Zombienet to access the configuration file and other files in the current directory. If you want to mount a different directory, replace `$(pwd)` with the desired directory path.
 
 
 ## Providers
 
-Zombienet is a JavaScript library designed to run on Node.js and support different backend providers in running the nodes. At this moment [Kubernetes](https://kubernetes.io/){target=_blank}, [Podman](https://podman.io/){target=_blank} and, native are supported.
+Zombienet is a JavaScript library designed to run on Node.js and support different backend providers in running the nodes. At this moment, [Kubernetes](https://kubernetes.io/){target=_blank}, [Podman](https://podman.io/){target=_blank}, and local are supported.
```

Comment:
I don't think mentioning that Zombienet is a Javascript library is necessary in this section. You could add that to the introduction instead